PWGCF/JCorran/Core/JCORRANLinkDef.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ class JFFlucAnalysis + ;
+#pragma link C++ class AliJHistManager + ;
```

Comment:
hello, is it really necessary to change all names of our classes? We have to waste our time for nothing. Can you explain why this rule was in place?

Reply:
I understand this is a little work but we do not have AliPhysics anymore and the Ali refers to this. Also for O2 there should not be O2 in the class names.
Have you already tried to use Visual Studio Code for it? It has a very good "replace in multiple files" feature.

Reply:
The codes don't matter if they are in the AliPhysics or O2, they are just classes that are used on other occasions, like toy mc etc. If this is the rule enforced for o2, we can change them. We were just forced to add the prefix.

Reply:
done!

Reply:
@dongjokim I don't understand your comment about where the code is. 
The rule which has been agreed and followed with all code in O2Physics up to now is that we do not keep the `Ali` prefix. E.g. AliFemto had to do a major work. I think the same should apply to your code. You write "done" but there are still many classes with Ali. What is your worry here, the additional work?
We can also involve @ddobrigk if needed here as analysis coordinator.

I anyway still have to see with @jaelpark how to refactor the code which copies the track values. Unfortunately, we were not able to discuss this last week due to reasons outside of our control...

Reply:
ok, thanks. I meant for the naming inside the code as well as the code naming. It looks like this can confuse people. Now I have changed the inner class's naming as well.

Reply:
Great - thanks! I will see with @jaelpark about the copying and then get back to you.